flytekit/clis/sdk_in_container/register.py

```diff
@@ -113,6 +108,14 @@
     is_flag=True,
     help="Execute registration in dry-run mode. Skips actual registration to remote",
 )
+@click.option(
+    "--activate-launchplans",
+    "--activate-launchplan",
+    default=False,
+    is_flag=True,
+    help="Flip current active version to the newly registered version of an active LaunchPlan. This combines to "
```

Comment:
typo: to -> two

Reply:
Consider rephrasing this, something like: "Activate newly registered Launchplans. This operation deactivates previous versions of Launchplans."

Reply:
Done